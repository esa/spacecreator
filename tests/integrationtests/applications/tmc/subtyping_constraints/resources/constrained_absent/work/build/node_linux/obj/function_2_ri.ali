V "GNAT Lib v8"
A -gnatwa
A -O0
A -Wall
A -Wextra
A -gnatA
A -fdata-sections
A -ffunction-sections
A -gnatd.E
A -fstack-usage
A -g
A -fstack-check=specific
A -mtune=generic
A -march=x86-64
P ZX

RN
RV NO_IMPLEMENTATION_PRAGMAS
RV SPARK_05

U function_2_ri%b	function_2_ri.adb	f1e5fa78 NE OO PK

U function_2_ri%s	function_2_ri.ads	63d0af23 EE NE OO PK
W adaasn1rtl%s		adaasn1rtl.adb		adaasn1rtl.ali
W constrained_absent_dataview%s  constrained_absent_dataview.adb  constrained_absent_dataview.ali
W interfaces%s		interfac.ads		interfac.ali
W interfaces.c%s	i-c.adb			i-c.ali
W interfaces.c.strings%s  i-cstrin.adb		i-cstrin.ali
W taste_basictypes%s	taste_basictypes.adb	taste_basictypes.ali

D ada.ads		20170911084841 76789da1 ada%s
D a-charac.ads		20170908151217 2d3ec45b ada.characters%s
D a-chlat1.ads		20170908151217 66457d31 ada.characters.latin_1%s
D a-unccon.ads		20170908151217 0e9b276f ada.unchecked_conversion%s
D adaasn1rtl.ads	20230103105717 ba1d55d4 adaasn1rtl%s
D board_config.ads	20230103105717 ff248fc1 board_config%s
D constrained_absent_dataview.ads  20230103105717 ecfc98a1 constrained_absent_dataview%s
D function_2_ri.ads	20230103105703 11029cea function_2_ri%s
D function_2_ri.adb	20230103105703 e0e76692 function_2_ri%b
D interfac.ads		20180111085525 5ab55268 interfaces%s
D i-c.ads		20170908151217 5421119b interfaces.c%s
D i-cstrin.ads		20180111085525 341d0ddf interfaces.c.strings%s
D system.ads		20180111085525 4635ec04 system%s
D s-bitops.ads		20180111085525 0765e3a3 system.bit_ops%s
D s-exctab.ads		20180111085525 54135002 system.exception_table%s
D s-parame.ads		20180111085525 f896c45c system.parameters%s
D s-stalib.ads		20180111085525 09bd3940 system.standard_library%s
D s-unstyp.ads		20180111085525 34867c83 system.unsigned_types%s
D taste_basictypes.ads	20230103105717 102ef363 taste_basictypes%s
X 5 adaasn1rtl.ads
5K9*adaasn1rtl 125e15 8|6w6 13r37
19B12*Asn1Boolean{boolean} 8|13r48
X 7 constrained_absent_dataview.ads
26K9*CONSTRAINED_ABSENT_DATAVIEW 134e32 8|4w6 4r39
75R6*asn1SccMyChoice 83e11 8|11r32 9|7r20 9r23
X 8 function_2_ri.ads
9K9*Function_2_RI 19l5 19e18 9|4b14 17l5 17t18
11U14*Pi_2 11=20 9|6b14 15l8 15t12
11r20 P1{7|75R6} 9|7b8 14m18
13U14*Check_Queue 13<27
13b27 Res{5|19E12}
17V13*To_C_Pointer{12|47P9} 17>27
17a27 State_As_String{string} 18r20
X 9 function_2_ri.adb
8U17*C_Pi_2 9=11 14s10
9r11 P1{7|75R6}
X 10 interfac.ads
38K9*Interfaces 8|7r6 7r32 10|184e15
X 11 i-c.ads
18K20*C 8|7r17 7r43 11|230e17
X 12 i-cstrin.ads
36K22*Strings 8|7w19 7r45 12|106e25
47P9*chars_ptr 8|17r60
60V13*New_String{47P9} 8|18s8
X 19 taste_basictypes.ads
26K9*TASTE_BasicTypes 8|5w6 5r39 19|95e21

