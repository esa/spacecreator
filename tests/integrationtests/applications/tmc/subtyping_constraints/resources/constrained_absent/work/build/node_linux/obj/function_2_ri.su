function_2_ri.ads:17:4:To_C_Pointer	32	static
function_2_ri.adb:6:4:Pi_2	32	static
