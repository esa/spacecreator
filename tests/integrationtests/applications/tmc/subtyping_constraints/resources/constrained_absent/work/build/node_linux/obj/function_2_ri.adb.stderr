function_2_ri.ads:5:06: warning: no entities of "Taste_Basictypes" are referenced
function_2_ri.ads:5:35: warning: use clause for package "TASTE_BasicTypes" has no effect
function_2_ri.ads:13:27: warning: "Check_Queue.Res" is an 8-bit Ada Boolean
function_2_ri.ads:13:27: warning: use appropriate corresponding type in C (e.g. char)
